api.md

```diff
@@ -1832,29 +1832,75 @@ DROP MATERIALIZED VIEW contagg_view;
 
 ## refresh_continuous_aggregate() :community_function: [](refresh_continuous_aggregate)
 
-Refresh all buckets of a continuous aggregate between two points of
-time. 
+Refresh all buckets of a continuous aggregate in the _refresh window_
+given by `window_start` and `window_end`.
+
+A continuous aggregate materializes aggregates in time buckets (e.g.,
+min, max, average over 10 minutes of data), as determined by the
+continuous aggregate view query's `time_bucket` expression. Therefore,
+when refreshing the continuous aggregate, only buckets that completely
+fit within the refresh window will be refreshed. In other words, it is
+not possible to compute the aggregate over, e.g., half a bucket, and
+any buckets that do no fit within the given refresh window will be
+excluded.
+
+The function expects the window parameter values to have a time type
+that is compatible with the continuous aggregate's time bucket
+expression (e.g., if the time bucket is specified in `TIMESTAMP WITH
+TIME ZONE`, then the start and end time supplied should be a date or
+timestamp type). Note that a continuous aggregate using the `TIMESTAMP
+WITH TIME ZONE` type aligns with the UTC time zone, so any potential
+time zone shifts need to be accounted for when refreshing.
 
-The function expects the parameter values to have the same time type
-as used in the continuous aggregate's time bucket expression (e.g., if
-the time bucket specifies in `timestamptz`, then the start and end time
-supplied should also be `timestamptz`).
 
 #### Required Arguments [](refresh_continuous_aggregate-required-arguments)
 
 |Name|Description|
 |---|---|
 | `continuous_aggregate` | (REGCLASS) The continuous aggregate to refresh. |
-| `window_start` | Start of the window to refresh, has to be before `window_end`. `NULL` is eqivalent to `MIN(timestamp)` of the hypertable. |
-| `window_end` | End of the window to refresh, has to be after `window_start`. `NULL` is eqivalent to `MAX(timestamp)` of the hypertable. |
+| `window_start` | Start of the window to refresh (inclusive), has to be before `window_end`. `NULL` is equivalent to an open-ended start of the window. |
+| `window_end` | End of the window to refresh (exclusive), has to be after `window_start`. `NULL` is equivalent to open-ended end of the window. |
+
+#### Errors
+
+An error is given if:
+
+- The refresh window does not cover at least one full bucket. Note
+  that, even though the window is one bucket in size, it might not
+  align with the start and the end of a bucket in the continuous
+  aggregate.
+- The function is called in a transaction block.
+  
 
 #### Sample Usage [](refresh_continuous_aggregate-examples)
 
-Refresh the continuous aggregate `conditions` between `2020-01-01` and
-`2020-02-01` exclusive.
+A continuous aggregate `conditions` uses the `TIMESTAMP WITH TIME
```

Comment:
`conditions` is quite common name for hypertables in our examples, thus it is strange to see as the name of a continuous aggregate.  Examples in the create cagg uses `continuous_aggregate_view`.
It is quite common to use the bucket size in the name of a cagg. May be `conditions_cagg_1d`? In such case it would be good to fix the create cagg example to use the same name.

Reply:
I have used `conditions` for hypertables and `conditions_summary` (`conditions_weekly_summary`,  `conditions_monthly_summary`, etc.) for various continuous aggregates.